doc/mgr/dashboard.rst

```diff
@@ -1070,3 +1070,162 @@ and loosely coupled fashion.
 
 .. include:: dashboard_plugins/feature_toggles.inc.rst
 .. include:: dashboard_plugins/debug.inc.rst
+
+
+Troubleshooting the Ceph Dashboard
+----------------------------------
+
+Locating the Ceph Dashboard
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are unsure of the location of the Ceph Dashboard, run the following command::
+
+    $ ceph mgr services | grep dashboard
+    "dashboard": "https://host:port"
+
+The command returns the URL where the Ceph Dashboard is located: ``https://<host>:<port>/``
+
+
+Accessing the Ceph Dashboard
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are unable to access the Ceph Dashboard, run through the following
+commands:
+
+#. Verify the Ceph Dashboard module is enabled::
+
+    $ ceph mgr module ls
+
+#. Ensure the Ceph Dashboard module is listed in the ``enabled_modules``
+   section. Example snipped output from the ``ceph mgr module ls`` command::
+
+    {
+      "always_on_modules": [
+        "balancer",
+        "crash",
+        "devicehealth",
+        "orchestrator",
+        "osd_support",
+        "pg_autoscaler",
+        "progress",
+        "rbd_support",
+        "status",
+        "telemetry",
+        "volumes"
+      ],
+      "enabled_modules": [
+        "dashboard",
+        "iostat",
+        "restful"
+      ],
+      "disabled_modules": [
+        {
+          ...
+        }
+      ]
+    }
+
+#. If it is not listed, activate the module with the following command::
+
+    $ ceph mgr module enable dashboard
+
+#. Check the Ceph Dashboard and/or mgr log file for any errors::
+
+    $ Path to logfile(s)
+
+#. Ensure the SSL/TSL support is configured properly:
+
+   * Check if the SSL/TSL support is enabled::
+
+       $ ceph config get mgr mgr/dashboard/ssl
+
+   * If the command returns ``true``, verify a certificate exists::
+
+       $ License file check
+
+   * If not, run to generate a self-signed certificate::
+
+       $ ceph dashboard create-self-signed-cert
+
+
+Trouble Logging into the Ceph Dashboard
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are unable to log into the Ceph Dashboard and you receive the following
+error, run through the procedural checks below:
+
+.. image:: images/dashboard/invalid-credentials.png
+
+#. Check that your user credentials are correct. If you are seeing the
+   notification message above when trying to log into the Ceph Dashboard, it
+   is likely you are using the wrong credentials. Double check your username
+   and password, and ensure the caps lock key is not enabled by accident.
+
+#. If your user credentials are correct, but you are experiencing the same
+   error, check that the user account exists::
+
+    $ ceph dashboard ac-user-show <username>
+
+   This command returns your user data. If the user does not exist, it will
+   print::
+
+    $ Error ENOENT: User <username> does not exist
+
+#. Check if the user is enabled::
+
+    $ ceph dashboard ac-user-show <username> {..., "enabled": true, ...}
```

Comment:
I don't know how, but i think we should mark or mention that `{..., "enabled": true, ...}` is the output of the given command. Currently it looks more that this is part of the command line/arguments.

Reply:
Thanks for the comment. It was supposed to be in two different lines. I forgot the line break.